cli/dts/lib.deno.unstable.d.ts

```diff
@@ -1181,6 +1181,35 @@ declare namespace Deno {
    * then the underlying HttpConn resource is closed automatically.
    */
   export function serveHttp(conn: Conn): HttpConn;
+
+  /** **UNSTABLE**: New option, permissions are yet to be vetted.
+   *
+   * Use the peermissions options to change the level of access the test will
+   * have. By default it will inherit the permissions of its parent thread. The permissions
+   * of a test can't be extended beyond its parent's permissions reach.
+   * - "inherit" will take the permissions of the thread the test is created in
+   * - You can disable/enable permissions all together by passing a boolean
+   * - You can provide a list permissions to narrow the permissions a test has.
+  */
+  export interface TestDefinition {
+    /** Set to `"none"` to disable all the permissions in the test. */
+    permissions?: "inherit" | "none" | {
```

Comment:
I think the "inherit" variant is redundant, because `undefined` means the same thing.

Reply:
I see that `WorkerOptions.permissions` has the same redundancy, so I suppose it's is not getting worse.

Reply:
Initially this was its own much simpler thing with just `boolean | []string` but noticed we had permissions in workers so adopted from the permission options workers (also re-using the same parser).

I think the options need to be refined but the intention is to keep the two sets of options the same and aligned (test permissions being aligned to worker permissions).

Changing the definition of worker permissions would be a breaking change so I want to punt that discussion down the line a bit.

Reply:
> Changing the definition of worker permissions would be a breaking change so I want to punt that discussion down the line a bit.

They are unstable, are they not?